index.js

```diff
@@ -61,7 +87,7 @@ module.exports = {
     // @author https://github.com/locobert
     // heavily based on vueify (Copyright (c) 2014-2016 Evan You)
     const { script, template } = vueCompiler.parseComponent(src, { pad: true});
-    const transformedScript = transformBabel(script.content);
+    const transformedScript = script ? transforms[script.lang || 'babel'](script.content) : '';
```

Comment:
My understanding here is that if <script> does not exist in the template, it gets replaced by an empty string. Perhaps it's just overthinking it.